README.md

```diff
@@ -27,9 +32,19 @@ mkdir myApp && cd myApp
 yo react-zeal
 
 # install all the things
+
+# with yarn
+yarn
+
+# with npm
 npm install
 
 # start your app on http://localhost:3000
+
+# with yarn
+yarn start
+
+# with npm
```

Comment:
Here, I'm not sure we want to give the option.  We use Yarn for our generated projects.  I guess there's no reasons someone couldn't choose to use npm, but that's not our recommendation.  How opinionated do we want to be?